pkg/test/builder/unstructured.go

```diff
@@ -94,7 +95,7 @@ func UnstructuredV1beta1TR(taskrun *v1beta1.TaskRun, version string) *unstructur
 	}
 }
 
-func UnstructuredV1beta1T(task *v1beta1.Task, version string) *unstructured.Unstructured {
+func UnstructuredT(task *v1beta1.Task, version string) *unstructured.Unstructured {
```

Comment:
I wonder if we should rename this 🤔 
we are approaching pipelines API version v1 so probably we can use `UnstructuredT` for that. WDYT?

Reply:
For now, i did this because there will be only 1 version, we can handle more later at that time